code/game/machinery/cryopod.dm

```diff
@@ -574,9 +572,9 @@
 	var/obj/machinery/cryopod/target_cryopod = null
 	if(free_cryopods.len)
 		target_cryopod = safepick(free_cryopods)
+		var/turf/T = get_turf(person_to_cryo)
```

Comment:
What was the reason for moving this up? It won't be used if the check below fails.

Reply:
Whoops, moving it back.

Reply:
Ok that makes sense.